actions.py

```diff
@@ -121,6 +137,36 @@ def co2_kg_interpolation(distance_km, flight_class, include_rfi=True):
 
         return co2_kg_per_km * distance_km
 
+    @staticmethod
+    def co2_kg_climatiq(distance_km, flight_class, departure_iata, destination_iata):
+        if distance_km < 760:  # longest UK domestic flight (Inverness<->Gatwick)
+            emission_factor = CLIMATIQ_EMISSION_FACTORS["domestic"]
+        elif (
+            distance_km < 3700
```

Comment:
Should we have these as constants somewhere?

Reply:
(as in, the two distance thresholds)

Reply:
Yeah, that'd be better. Thanks for the suggestion 🙂